tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -287,6 +289,23 @@ void ordering_test_cases() {
         std::forward_list<SynthOrdered> b = {10, 20, 40};
         ordered_containers_test(a, a, b);
     }
+    { // slice
+        std::valarray<int> a(20);
+        for (int i = 0; i < 12; ++i) {

```

Comment:
Why are only the first 12 elements being assigned ascending values, while the last 8 elements remain `0`?